Row.h

```diff
@@ -142,12 +142,12 @@ void Row_setPidColumnWidth(pid_t maxPid);
 /* Sets the size of the UID column based on the passed UID */
 void Row_setUidColumnWidth(uid_t maxUid);
 
+/* Takes number in kibibytes (base 1024). Prints 6 columns. */
+void Row_printKBytes(RichString* str, unsigned long long number, bool coloring);
+
 /* Takes number in bytes (base 1024). Prints 6 columns. */
 void Row_printBytes(RichString* str, unsigned long long number, bool coloring);
 
-/* Takes number in kilo bytes (base 1024). Prints 6 columns. */
-void Row_printKBytes(RichString* str, unsigned long long number, bool coloring);
-
```

Comment:
Why switch the order of these two declarations?

Reply:
The .c file swaps the definitions of two functions as I now make `Row_printBytes` call `Row_printKBytes` rather than the other way around.